File: Selection-Sort/python/simple.py
import random;

arr = [random.randint(-20,50) for _ in range(10)]

print(arr)


def sort():
    for i in range(0,len(arr)-1):
        sortedLastIndex = i;
        for j in range(i+1,len(arr)):
            if arr[sortedLastIndex] > arr[j]:
                sortedLastIndex = j

        if sortedLastIndex != i:
            tmp = arr[i]
            arr[i] = arr[sortedLastIndex]
            arr[sortedLastIndex] = tmp

sort();

print(arr)

File: Selection-Sort/js/simple.js
const arr = []

function initData(count){
    for(let i = 0 ; i < count ; i++){
        arr.push(Math.floor(Math.random()*50-20))
    }
}

initData(10)

console.log(arr)

function sort(){
    let sortedLastIndex = 0 , 
        flag = false,
        tmp;
    for(let i = 0 ; i < arr.length - 1; i++){
        sortedLastIndex = i
        for(let j = i + 1; j < arr.length ; j++){
            if(arr[sortedLastIndex] > arr[j]){
                sortedLastIndex = j
                flag = true
            }
        }
        if(flag){
            tmp = arr[i] 
            arr[i] = arr[sortedLastIndex]
            arr[sortedLastIndex] = tmp
            flag = false;
        }
    }
}

sort()

console.log(arr)